package.json

```diff
@@ -41,7 +41,7 @@
     "lint": "npx npm-run-all -p -c -l lint:*"
   },
   "dependencies": {
-    "fast-glob": "^2.2.4",
+    "fast-glob": "^2.2.6",
```

Comment:
`tiny-glob` seems to have won benchmarks, maybe we update to that in the future.

also, why isn't this coming through Renovate? is there something wrong with Renovate?

Reply:
I'll look into that. Closing as `master` is now good with the other PR merged.